TASK/TaskRappelDownWall.md

```diff
@@ -5,22 +5,22 @@ ns: TASK
 
 ```c
 // 0xEAF66ACDDC794793
-void _TASK_RAPPEL_DOWN_WALL(Any p0, Any p1, Any p2, Any p3, Any p4, Any p5, Any p6, Any p7, Any p8, Any p9, Any p10);
+void _TASK_RAPPEL_DOWN_WALL(Ped ped, float x1, float y1, float z1, float x2, float y2, float z2, float heading, int ropeID, char* Clipset, 1);
```

Comment:
Is the `1` at the end a typo, or did you intend to have another parameter there? It should always be a type followed by a parameter name, where the name doesn't start with a number.